docs/G-Codes.md

```diff
@@ -60,6 +60,12 @@ Klipper also supports the following standard G-Code commands if the
 - Set SD position: `M26 S<offset>`
 - Report SD print status: `M27`
 
+In addition, the following extended commands are availble when the
+"virtual_sdcard" config section is enabled.
+- List all gcode files (including those in subdirectories): `GET_FILE_LIST`
+- Load a file and start SD print: `PRINT_FILE_START FILENAME=<filename>`
+- Unload File and Clear SD state:  `SDCARD_RESET_FILE`
```

Comment:
I think it would be preferable if we could use a common prefix for all the commands of the module.  For example, "SDCARD_LIST_FILES,SDCARD_PRINT_FILE,SDCARD_RESET_FILE" or "PRINT_FILE_LIST,PRINT_FILE_START,PRINT_FILE_REST", or something similar.

Reply:
Ok, I will change to SDCARD_LIST_FILES and SDCARD_PRINT_FILE.  I think that verbiage is most clear on what it does.